codec/console/dec/src/h264dec.cpp

```diff
@@ -479,8 +479,6 @@ int32_t main (int32_t iArgC, char* pArgV[]) {
   }
 
 
-
-
```

Comment:
Unnecessary whitespace changes here in h264dec.cpp

Reply:
Thanks, restored to original state.